hack/errata.py

```diff
@@ -90,6 +92,58 @@ def notify(message, webhook=None):
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    pulls = repo.get_pulls(state='open', sort='created', base='master')
+    for pr in pulls:
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr.number, pr.body
+        except Exception as e:
```

Comment:
Nit: the `yield` does not need to be covered by this parsing-focused try/except.

Reply:
we don't want to yield if exception occurred